lib/Driver/WindowsToolChains.cpp

```diff
@@ -186,6 +190,21 @@ toolchains::Windows::constructInvocation(const DynamicLinkJobAction &job,
   context.Args.AddAllArgs(Arguments, options::OPT_linker_option_Group);
   context.Args.AddAllArgValues(Arguments, options::OPT_Xclang_linker);
 
+  switch (context.OI.LTOVariant) {
+  case OutputInfo::LTOKind::LLVMThin:
+  case OutputInfo::LTOKind::LLVMFull: {
+    if (Linker.empty())
+      Arguments.push_back("-fuse-ld=lld");
+    if (context.OI.LTOVariant == OutputInfo::LTOKind::LLVMThin) {
+      Arguments.push_back("-flto=thin");
+    } else {
+      Arguments.push_back("-flto=full");
+    }
+    break;
+  }
+  case OutputInfo::LTOKind::None: break;
```

Comment:
This is another example where you haven't used git-clang-format. Please do so before this lands!